ext/files.ts

```diff
@@ -2,119 +2,136 @@
 import { R2Bucket } from 'https://cdn.jsdelivr.net/npm/@cloudflare/workers-types@4.20230821.0/index.ts'
 import { join } from 'https://deno.land/std@0.199.0/path/mod.ts'
 import { createExtension } from '../extensions.ts'
+import { AppContext } from '../mod.ts'
 
 // An extension to serve static files from Cloudflare R2, an S3 bucket, or the local file system.
 
+type FilesCommonType = {
```

Comment:
replace `FilesCommonTypes` with `GeneralOptions` and any `Type` in type name with the word `Options`

Reply:
we know it's a type, so it's unnecessary to make the user aware of it